portal-web/docroot/html/taglib/ui/page_iterator/start.jsp

```diff
@@ -410,6 +414,77 @@ NumberFormat numberFormat = NumberFormat.getNumberInstance(locale);
 	}
 </script>
 
+<script data-senna-track="temporary" type="text/javascript">
+	var dropdown = document.getElementById("<%= ariaPagination %>");
+
+	var button = dropdown.querySelector('.dropdown-toggle');
+	var list = dropdown.querySelector('.dropdown-menu');
+
+	var options = list.querySelectorAll('.dropdown-item');
+	var selectedItemValue = button.dataset.attribute;
+
+	function onButtonKeyDown(event) {
+		if (event.key === 'ArrowDown' || event.key === 'ArrowUp' || event.key === 'Enter' || event.key === 'Space') {
+			event.preventDefault();
+
+			list.classList.add('show');
+			button.setAttribute('aria-expanded', 'true');
+
+			var selectedOption = list.querySelector('.active');
+
+			if (selectedOption) {
+				selectedOption.focus();
+			}
+		}
+	}
+
+	button.addEventListener('keydown',onButtonKeyDown );
+
+	function onLeaveDropdown() {
+		list.classList.remove('show');
+		button.setAttribute('aria-expanded', 'false');
```

Comment:
These should be sorted or there should be an empty line between them to indicate that the order matters.

Reply:
All fixed.